components/nimbus/src/evaluator.rs

```diff
@@ -37,6 +40,7 @@ pub struct TargetingAttributes {
     pub days_since_install: Option<i32>,
     pub days_since_update: Option<i32>,
     pub active_experiments: HashSet<String>,
+    pub event_store: Option<EventStore>,
```

Comment:
This doesn't look right here. `TargetingAttributes` are the thing that is serialized into JSON for direct access from JEXL; but `event_store` is accessed through a transform.

Would you consider a different way of moving the event store, or not serializing it on `TargetingAttibutes`?

Reply:
This was something @jeddai  and I discussed but we never settled on a good way of getting the event store into the JEXL evalutor. If this can be improved, I'm all for it.

Reply:
It is now a parameter that is passed in all the way down the chain. Seems to work well!